server/util.go

```diff
@@ -52,9 +52,12 @@ type Context struct {
 }
 
 func NewDefaultContext() *Context {
+	cmtCfg := cmtcfg.DefaultConfig()
+	// switch to priority as CAT breaks the e2e tests here
+	cmtCfg.Mempool.Type = cmtcfg.MempoolTypePriority
```

Comment:
use CAT has insane errors related to decoding / encoding that we should look into.